joomla/joomla-cms
# [4.0] Add period to the success message

### Testing Instructions
Go to System >Site Modules
Edit a module.
Click Save & Close button.
See success message.

Perform similar steps to Plugins where the success message ends with a period.

### Actual result BEFORE applying this Pull Request
Message has no period.


### Expected result AFTER applying this Pull Request
Message ends with a period.
